orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java

```diff
@@ -0,0 +1,328 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb;
+
+import akka.Done;
+import akka.actor.ActorSystem;
+import akka.stream.ActorMaterializer;
+import akka.stream.alpakka.orientdb.javadsl.*;
+import akka.stream.javadsl.Sink;
+import akka.testkit.JavaTestKit;
+import com.orientechnologies.orient.client.remote.OServerAdmin;
+import com.orientechnologies.orient.core.db.ODatabaseRecordThreadLocal;
+import com.orientechnologies.orient.core.db.OPartitionedDatabasePool;
+import com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx;
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import com.orientechnologies.orient.object.db.OObjectDatabaseTx;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.*;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+public class OrientDBTest {
+
+  private static OServerAdmin oServerAdmin;
+  private static OPartitionedDatabasePool oDatabase;
+  private static ODatabaseDocumentTx client;
+  private static ActorSystem system;
+  private static ActorMaterializer materializer;
+
+  //#init-settings
+  private static String url = "remote:127.0.0.1:2424/";
+  private static String dbName = "GratefulDeadConcerts";
+  private static String dbUrl = url + dbName;
+  private static String username = "root";
+  private static String password = "root";
+  //#init-settings
+
+  private static String source = "source1";
+  private static String sink1 = "sink1";
+  private static String sink2 = "sink2";
+  private static String sink3 = "sink3";
+
+  //#define-class
+  public class source1 {

```

Comment:
Is it intentional to have a non-capitalized class name?

Reply:
Yes, the corresponding orientdb class name is `source1`.